pkgs/top-level/all-packages.nix

```diff
@@ -14507,6 +14507,16 @@ with pkgs;
 
   lttng-ust = callPackage ../development/tools/misc/lttng-ust { };
 
+  lttng-ust-2-10 = (lttng-ust.override {
+    liburcu = liburcu-0-12;
+  }).overrideAttrs (oldAttrs: rec {
+    version = "2.10.5";
+    src = fetchurl {
+      url = "https://lttng.org/files/lttng-ust/${oldAttrs.pname}-${version}.tar.bz2";
```

Comment:
```suggestion
      url = "https://lttng.org/files/lttng-ust/lttng-ust-${version}.tar.bz2";
```

Reply:
Done. Included in `aa62a2f7926e01d534f0dfcd9e7701cb6721ea7a`.